script.js

```diff
@@ -0,0 +1,12 @@
+isTextUppercase('Hello'); // false
+isTextUppercase('HELLO'); // true
+
+
+function isTextUppercase(text) {
+    if (text === text.toUpperCase()){
+        result = true;
+    } else {
+        result = false;
+    }
+    console.log(result)
```

Comment:
console log is not equal to return, function should return boolean, you can simplify it - there is no need for an if...else statement